### ISSUE TITLE
Is it possible to compute the mAP only for specific regions in an image ?

### ISSUE THREAD

**Issue by YellowKyu** [cocodataset/cocoapi#249]
Hi folks !

Thanks for your amazing work, I have a little question here:
Let's say at testing (and training too actually), I want to ignore some region/area from the images I have to compute the mAP, how can I achieve that ? I can't just remove my ground truth or training bounding boxes that are located in those regions and then compute the mAP normally because it leads to big number of False Positive if bounding boxes are predicted in unwanted regions.

To sum up, is it possible to compute the mAP only for specific regions in an image ?

Many thanks !
